### PR TITLE
SSL Synthetics tests

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Follow up on https://github.com/DataDog/documentation/pull/4574

### Motivation
Fixing master conflict

### Preview link

https://docs-staging.datadoghq.com/gus/ssl-tests/synthetics/api_tests/?tab=httptest

### CODE REVIEW

File: content/en/synthetics/api_tests/_index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+## Overview
+
+API tests are HTTP requests (`GET`, `POST`, `PUT`, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals [from multiple locations around the world][1]. These checks verify that your applications are responding to requests, as well as that they meet any conditions you define—such as response time, HTTP status code, and header or body contents.
```
Comment (gabsn) [inline content/en/synthetics/api_tests/_index.md]: it is wrong. Maybe the rebase has messed up everything 🤔

File: content/en/synthetics/api_tests/_index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+## Network timings
+
+The Synthetics details page displays the following network timings:
+
+| Timing                    | Description                                                                                           |
+|---------------------------|-------------------------------------------------------------------------------------------------------|
+| `DNS`                       | Time spent resolving the DNS name of the last request.                                                |
+| `Connect`                   | Time spent establishing a connection to the server.                                                   |
+| `SSL`                       | Time spent for the TLS handshake. If the last request is not over HTTPS, this metric does not appear. |
```
Comment (gabsn) [inline content/en/synthetics/api_tests/_index.md]: missing the link to the ssl errors.

Comment (l0k0ms) [reply content/en/synthetics/api_tests/_index.md]: the link is above in the Test Failure section :)

File: content/en/synthetics/api_tests/_index.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+## Overview
+
+API tests are useful to help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests, as well as that they meet any conditions you define—such as response time, HTTP status code, and header or body contents. They are executed periodically from several locations around the world, the full list can be retrieved [through the Datadog API][1].
```
Comment (apigirl) [inline content/en/synthetics/api_tests/_index.md]:
```suggestion
API tests are useful to help you monitor your API endpoints and alert you when they are failing or too slow. These checks verify that your applications are responding to requests, as well as that they meet any conditions you define—such as response time, HTTP status code, and header or body contents. Use the[Datadog API][1] to see the full list.
```

File: content/en/synthetics/api_tests/_index.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+Define the `HTTP` or `SSL` request you want to be executed by Datadog:
+
+{{< tabs >}}
+
+{{% tab "HTTP Test" %}}
+
+{{< img src="synthetics/api_tests/make-http-request.png" alt="Make HTTP Request" responsive="true" style="width:80%;" >}}
+
+Define the request you want executed by Datadog:
```
Comment (apigirl) [inline content/en/synthetics/api_tests/_index.md]:
```suggestion
Define the request you want to be executed by Datadog:
```

**Pull request merged**